articles/daas/daas-sd.md

```diff
@@ -0,0 +1,124 @@
+---
+title: UKCloud Desktop as a Service Service Definition
+description: Provides an overview of what is provided by the UKCloud Desktop as a Service service
+services: daas
+author: shighmoor
+reviewer: shighmoor
+lastreviewed: 22/09/2020
+toc_rootlink: Service Information
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Service Definition
+toc_fullpath: Service Information/daas-sd.md
+toc_mdlink: daas-sd.md
+---
+
+# UKCloud Desktop as a Service Service Definition
+
+## What is UKCloud Desktop as a Service?
+
+UKCloud Desktop as a Service provides a platform to deliver, protect and manage desktops and apps whilst containing costs and ensuring that end users can work anytime, anywhere, across any device.
+
+UKCloud Desktop as a Service provides significant security and compliance benefits to your organisation, whilst increasing workforce enablement through smarter more flexible working. This aligns with government policy around smart working within new Government Hubs. UKCloud Desktop as a Service also enables the Bring Your Own Device (BYOD) policy and supports remote working, without compromising application availability, security, and compliance.
+
+## What the service can help you achieve
+
+- **Deliver both persistent and non-persistent Windows 10 desktops.** Supports both standard and complex requirements within your organisation.
```

Comment:
can you please change to read 'Deliver both dedicated and shared Windows 10 desktops" ... this is then consistent with how we describe options later in the document,

Reply:
@gmartinukcloudcom, made that change if you could just take a look and approve please!